opm/parser/eclipse/EclipseState/Tables/Tables.cpp

```diff
@@ -1085,6 +1085,22 @@ struct flat_props< PVTWRecord, 0 > {
     }
 };
 
+constexpr const char* pvcdo_err[] = {
```

Comment:
I just mean that something like

```c++
throw std::run_error("PVCDO reference pressure cannot be defaulted");
```

is much easier to comprehend than

```c++
throw std::run_error(pvcdo_err[0]);
```

and that a few duplicated strings for error messages are not a large maintenance burden. That said, if you think the second alternative is better, that's okay with me as well.